community/README.md

```diff
@@ -1,5 +1,5 @@
 # Knative Community
 
 For community information, see
-[Knative Community](https://github.com/knative/docs/tree/master/community) in
-the [Docs](https://github.com/knative/docs) repo.
+[Knative Contributing](https://www.knative.dev/contributing/) on
+the [https://www.knative.dev](https://www.knative.dev/docs) site.
```

Comment:
Why route to `/docs`? It has a lot more content and I find it more difficult to notice contributing in the top right corner. I would update to the homepage.
```suggestion
the [https://www.knative.dev](https://www.knative.dev/) site.
```

Reply:
Updated to point to Knative.dev.